dockerfiles/theia-dev/build.sh

```diff
@@ -28,11 +50,13 @@ then
     rm -f $CHE_THEIA_GENERATOR_PACKAGE
     echo "Building Che Theia generator"
     cd "${base_dir}"/../../generator/ && yarn && yarn pack --filename $CHE_THEIA_GENERATOR_PACKAGE_NAME
+    # Build the file in a way that it's agnostic of timestamp/user who is doing it
+    docker run --rm -it -v "${base_dir}/../../generator:/workdir" ${GNUTAR_IMAGE_NAME} sh -c 'mkdir -p unpacked && tar zxf eclipse-che-theia-generator.tgz -C unpacked && tar czf eclipse-che-theia-generator.tgz -C unpacked package --owner=che-theia --group=che-theia --sort=name --clamp-mtime --mtime="CET 2020-01-01 00:00:00" && rm -rf unpacked'
```

Comment:
add line breaks (`\`) for legibility, please

Reply:
hardcoded timestamp? `2020-01-01 00:00:00` ?

Reply:
yes, we don't care